src/app/payment-hub/request-to-pay/incoming-request-export/incoming-request-export.component.html

```diff
@@ -64,7 +61,18 @@
           name="startdate"
           placeholder="comma-separated-list"
           id="startdate"
-          (keyup)="arrayConvert($event)"
+        />
+      </mat-form-field>
+      <mat-form-field>
+        <mat-label>START TO</mat-label>
```

Comment:
"Start To" field name doesn't make any sense. Did you mean "End Date"? Please correct.

Reply:
Resolved